src/renderer/components/AircraftSection/index.tsx

```diff
@@ -127,12 +155,17 @@ const index: React.FC<Props> = (props: Props) => {
     useEffect(() => {
         const checkMsfsInterval = setInterval(checkIfMSFS, 500);
 
-        return () => clearInterval(checkMsfsInterval);
```

Comment:
Why?

Reply:
Oh yeah that was me trying to do something and forgot to revert it completely, because it wasn't obviously changing the code XD